pkg/controller/gittrack/gittrack_controller.go

```diff
@@ -492,6 +497,11 @@ func (r *ReconcileGitTrack) ignoreObject(u *unstructured.Unstructured) (bool, st
 		return false, "", err
 	}
 
+	// Ignore namespaced objects in namespaced-only mode
```

Comment:
I think this comment is not quite right?
```suggestion
	// Ignore non-namespaced objects in namespaced-only mode
```